pom.xml

```diff
@@ -1,4 +1,5 @@
-<!--
+exit
+        <!--
```

Comment:
This might be the issue causing the build failure. @Wandji69 could you please check this suggestion locally and see.
```suggestion
<!--
```